src/sass/_common/wallet_header.scss

```diff
@@ -0,0 +1,225 @@
+            display: flex;
+            align-items: center;
+            border-radius: $BORDER_RADIUS;
+            cursor: pointer;
+            gap: 16px;
+            justify-content: space-between;
+            padding: 13.5px 16px;
+
+            &:hover {
+                background: var(--state-hover);
+            }
+            &--active {
+                background: var(--state-active);
+            }
+        }
+        &-accounts {
+            &-list {
+                max-height: 450px;
+                border-bottom: 1px solid var(--general-hover);
+                padding: 4px 8px;
+                overflow-y: scroll;
+            }
+        }
+        &-balance {
+            line-height: 20px;
+            font-size: $FONT_SIZE_S;
+            font-weight: bold;
+        }
+        &--content {
+            display: flex;
+            flex-direction: column;
+            width: 100%;
+            font-weight: normal;
+            line-height: 14px;
+            font-size: $FONT_SIZE_XXS;
+        }
+        &-dropdown {
+            display: none;
+            position: absolute;
+            background-color: var(--general-main-1);
+            box-shadow: 0 8px 16px 0 $COLOR_LIGHT_BLACK_3;
+            top: calc(100% + 4px);
+            right: -32px;
+            width: 320px;
+            border-radius: 4px;
+
+            &--show {
+                display: block;
+            }
+        }
+        &-header {
+            display: flex;
+            justify-content: space-between;
+            align-items: center;
+            padding: 12px;
+            border-bottom: 1px solid var(--general-hover);
+
+            &--text {
+                font-size: $FONT_SIZE_S;
+                font-weight: bold;
+                line-height: 1.5;
+            }
+            img {
+                width: 16px;
+                height: 16px;
+            }
+        }
+        &-footer {
+            display: flex;
+            font-size: $FONT_SIZE_S;
+            justify-content: space-between;
+            padding: 16px;
+            line-height: 27px;
+
+            & > span {
+                cursor: pointer;
+            }
+        }
+        &-icon {
+            &--container {
+                align-items: center;
+                border-radius: 4px;
+                justify-content: center;
+                height: 40px;
+                width: 64px;
+            }
+        }
+    }
+}
+
+.mobile {
+    &__wallet-platform-switcher {
+        &-current {
+            height: 72px;
+            display: flex;
+            justify-content: center;
+            border-bottom: 1px solid var(--general-section-2);
+            align-items: center;
+            padding: 0 32px;
+            cursor: pointer;
+        }
+        &-container {
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            gap: 8px;
+        }
+        &-logo {
+            width: 148px !important;
```

Comment:
which style is overriding it? is it possible to move this style after the place where it gets overriden in order to avoid using `!important`? or can we use some other approach to increase this CSS selector's specificity (priority) for your element without using !important?  :)

Reply:
i forget but let me check

Reply:
this is handled! i removed this classname completely and handled the other classname with media query